storybook/.eslintrc.json

```diff
@@ -7,7 +7,10 @@
         "@typescript-eslint/no-explicit-any": "off",
         "@typescript-eslint/no-non-null-assertion": "off",
         "no-console": "off",
-        "@comet/no-other-module-relative-import": "off"
+        "@comet/no-other-module-relative-import": "off",
+        "no-restricted-imports": "off",
+        "@typescript-eslint/no-unused-vars": "off",
+        "unused-imports/no-unused-imports": "off"
```

Comment:
@johnnyomair because we still use react 17 for storybook, the react barrel import is still necessary i guess. I disabled no-unused-vars and no-unused-imports for the pipeline to run through. The alternative would be to add  (`// eslint-disable-next-line @typescript-eslint/no-unused-vars, unused-imports/no-unused-imports`) for every line, like i did for the handful imports in docs/ 

Is it planned to update storybook? Then the react imports should be removed and the rules can be deleted again.

<img width="778" alt="Screenshot 2024-12-09 at 13 49 15" src="https://github.com/user-attachments/assets/87aa4d15-91f4-47bd-9bab-957850b6cd0c">

Reply:
We've recently updated Storybook to 8.4. The new JSX Transform should already be supported since 6.1: https://storybook.js.org/blog/storybook-6-1/. So maybe we need to adapt the config somehow?

Reply:
here we go, storybook updated